### PR TITLE
Skorvald Combat Replay & New Consumables

### PR DESCRIPTION
**Pull request by Linkaaaaa**
Added Skorvald's Combat Replay covering:
- Horizon Strike
- Crimson Dawn
- Solar Bolt
- Radiant Fury
- Supernova
- Solar Cyclone
- Punishing Kick (Skorvald and Anomaly)
- Cranial Cascade (Skorvald and Anomaly)
- Mist Bomb (Skorvald and Anomaly)
- Solar Stomp (Anomaly)
- Mist Smash (Anomaly)

Added new foods to the consumables tab:
 - Plate of Spicy Moa Wings
 - Steak with Winterberry Sauce
 - Feast of Delectable Birthday Cake
 - Jerk Poultry and Nopal Flatbread Sandwich

Added missing ascended food IDs for:
 - Eggs Benedict With Mint Parsley Sauce
 - Spiced Peppercorn Cheesecake
 - Bowl Of Fruit Salad With Orange Clove Syrup
 - Peppercorn Spiced Eggs Benedict
 - Bowl Of Sesame Fruit Salad
 - Spiced Pepper Creme Brulee
 - Mint Creme Brulee
 - Mint Pear Cured Meat Flatbread
 - Orange Clove Cheesecake
 - Clove And Veggie Flatbread
 - Clove Spiced Eggs Benedict
 - Clove Spiced Creme Brulee
 - Clove SpicedPearAndCuredMeatFlatbread
 - Plate Of Clove Spiced Clear Truffle Ravioli
 - Plate Of Peppered Clear Truffle Ravioli
 - Plate Of Sesame Poultry Aspic
 - Plate Of Clove Spiced Coq Au Vin
 - Plate Of Beef Carpaccio With Salsa Garnish
 - Plate Of Clove Spiced Beef Carpaccio
 - Plate Of Beef Carpaccio With Mint Garnish
 
Added several missing Skill IDs for Skorvald, Arkk and Ai
Added 2 new parser helpers
Added new effect GUIDs

Notes:

- Solar Cyclone decoration has been taken as reference from Kralkatorrik's ground effects during the orb purification, the direction of the attacks isn't always displayed correctly and would like your review on it.
- Currently missing the Solar Bloom explosion, i believe it's not a skill casted by the agent.
- Currently missing the eye beam during last 33% phase.

### CODE REVIEW

File: GW2EIEvtcParser/EncounterLogic/Fractals/ShatteredObservatory/Skorvald.cs [hunk 1]
```diff
@@ -207,5 +208,369 @@ protected override List<ArcDPSEnums.TrashID> GetTrashMobsIDs()
...
+                        int interval = 300;
+                        int attackEnd = start + duration;
+
+                        EffectGUIDEvent kick = log.CombatData.GetEffectGUIDEvent(EffectGUIDs.KickGroundEffect);
+                        if (kick != null)
+                        {
+                            IEnumerable<EffectEvent> kickEffects = log.CombatData.GetEffectEventsByEffectID(kick.ContentID).ToList().Where(x => x.Time >= start && x.Time < start + duration);
+                            foreach (EffectEvent kickEffect in kickEffects)
+                            {
+                                replay.Decorations.Add(new RotatedRectangleDecoration(true, 0, 300, (int)target.HitboxWidth, RadianToDegreeF(kickEffect.Orientation.Z), 0, (start, start + interval), "rgba(255, 0, 0, 0.2)", new PositionConnector(kickEffect.Position)));
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EncounterLogic/Fractals/ShatteredObservatory/Skorvald.cs:380]: Any example as to how it looks like vs how it is supposed to look like? Also shouldn't the decoration be a pie?

Comment (EliphasNUIT) [reply GW2EIEvtcParser/EncounterLogic/Fractals/ShatteredObservatory/Skorvald.cs:380]: Perhaps I'm confusing this with something else, I remember this attack being conicals

Comment (Linkaaaaa) [reply GW2EIEvtcParser/EncounterLogic/Fractals/ShatteredObservatory/Skorvald.cs:380]: https://youtu.be/QOe4nBTDUp8 at exactly 4:00 you can see which attack im referring to
Also just noticed as i scrolled through the vod that the 4th anomaly attack had 5 kick effects... I need to check that out tomorrow

**Comment by EliphasNUIT** [comment]
Thanks!

**Pull request merged**